cmd/lncli/commands.go

```diff
@@ -101,7 +101,7 @@ func actionDecorator(f func(*cli.Context) error) func(*cli.Context) error {
 			// two commands.
 			if s.Code() == codes.Unimplemented &&
 				(c.Command.Name == "create" ||
-					c.Command.Name == "unlock") {
+					c.Command.Name == "unlock" || c.Command.Name == "changepassword" ) {
```

Comment:
lines should always wrap at 80 chars. so pls put this new `||` on the following line.

Also, I think we want to handle all the wallet unlocker commands here. So how about we also add the `createwatchonly` case here while we are at it :)